src/keyboard.rs

```diff
@@ -1393,6 +1801,635 @@ pub enum Key<'a> {
     F35,
 }
 
+impl<'a> Key<'a> {
+    pub fn from_key_attribute_value(kav: &'a str) -> Self {
+        match kav {
+            // TODO: Report this in a better way.
+            "Unidentified" => Key::Unidentified(NativeKeyCode::Web("Unidentified")),
+            "Dead" => Key::Dead(None),
+            "Alt" => Key::Alt,
+            "AltGraph" => Key::AltGraph,
+            "CapsLock" => Key::CapsLock,
+            "Control" => Key::Control,
+            "Fn" => Key::Fn,
+            "FnLock" => Key::FnLock,
+            "NumLock" => Key::NumLock,
+            "ScrollLock" => Key::ScrollLock,
+            "Shift" => Key::Shift,
+            "Symbol" => Key::Symbol,
+            "SymbolLock" => Key::SymbolLock,
+            "Hyper" => Key::Hyper,
+            "Meta" => Key::Super,
+            "Enter" => Key::Enter,
+            "Tab" => Key::Tab,
+            "Space" => Key::Space,
```

Comment:
Oops! Space is encoded as `" "` on the web. See https://www.w3.org/TR/uievents-key/#keys-whitespace

Reply:
I guess that's what happens when I do bulk edits.